test/e2e/e2e_test.go

```diff
@@ -182,4 +182,33 @@ var _ = Describe("Workload cluster creation", func() {
 			}, result)
 		})
 	})
+
+	Context("Creating a single control-plane cluster", func() {
+		It("Should create a cluster with 1 worker node and can be scaled", func() {
+			By("Initializes with 1 worker node")
+			clusterctl.ApplyClusterTemplateAndWait(ctx, clusterctl.ApplyClusterTemplateAndWaitInput{
+				ClusterProxy: bootstrapClusterProxy,
+				ConfigCluster: clusterctl.ConfigClusterInput{
+					LogFolder:                clusterctlLogFolder,
+					ClusterctlConfigPath:     clusterctlConfigPath,
+					KubeconfigPath:           bootstrapClusterProxy.GetKubeconfigPath(),
+					InfrastructureProvider:   clusterctl.DefaultInfrastructureProvider,
+					Flavor:                   clusterctl.DefaultFlavor,
+					Namespace:                namespace.Name,
+					ClusterName:              clusterName,
+					KubernetesVersion:        e2eConfig.GetVariable(KubernetesVersion),
+					ControlPlaneMachineCount: pointer.Int64Ptr(1),
+					WorkerMachineCount:       pointer.Int64Ptr(1),
+					ClusterctlVariables: map[string]string{
+						"GCP_SUBNET_NAME": "capg-test-subnet",
```

Comment:
Do we need to create this subnet (`capg-test-subnet`) before running the test? Guessing so......

Reply:
No, we don't need to create the subnet. We just need to specify the name and CIDR range. capg operator will create the subnet.